src/main/java/com/acmerobotics/robomatic/util/TeleOpActionImpl.java

```diff
@@ -0,0 +1,8 @@
+package com.acmerobotics.robomatic.util;
+
+import com.acmerobotics.robomatic.util.StickyGamepad;
+import com.qualcomm.robotcore.hardware.Gamepad;
+
+public interface TeleOpActionImpl {
```

Comment:
Impl is short for Implementation, this is an interface

Reply:
If you had some abstract interface and then a default implementation of that interface, you would call the implementation interfacenameImpl